golos.publication/golos.publication.abi

```diff
@@ -614,6 +624,96 @@
             "type": "limitsarg"
         }
     ]
+  },
+  {
+    "name": "st_max_vote_changes",
+    "base": "",
+    "fields": [
+      {
+        "name": "max_vote_changes",
+        "type": "uint8"
+      }
+    ]
+  },
+  {
+    "name": "st_cashout_window",
+    "base": "",
+    "fields": [
+      {
+        "name": "cashout_window",
+        "type": "uint32"
+      }
+    ]
+  },
+  {
+    "name": "st_upvote_lockout",
+    "base": "",
+    "fields": [
+      {
+        "name": "upvote_lockout",
+        "type": "uint8"
+      }
+    ]
+  },
+  {
+    "name": "st_max_beneficiaries",
+    "base": "",
+    "fields": [
+      {
+        "name": "max_beneficiaries",
+        "type": "uint8"
+      }
+    ]
+  },
+  {
+    "name": "st_max_comment_depth",
+    "base": "",
+    "fields": [
+      {
+        "name": "max_comment_depth",
+        "type": "uint16"
+      }
+    ]
+  },
+  {
+    "name": "posting_state",
+    "base": "",
+    "fields": [
+      {
+        "name": "id",
+        "type": "uint64"
+      },
+      {
+        "name": "max_vote_changes_param",
```

Comment:
for me, it's not convenient to have `_param` suffix in field names of the state structure, we already know there are params in it